### PR TITLE
[TINKERPOP-3025] Add missing l_trim() and r_trim() to python

### PR DESCRIPTION
**Pull request by ryn5**
https://issues.apache.org/jira/projects/TINKERPOP/issues/TINKERPOP-3025?filter=allopenissues Added missing l_trim/r_trim, deprecated ltrim/rTrim, and minor consistency fixes to python.

### CODE REVIEW

**Comment by spmallette** [comment]
needs a changelog entry but this was the right idea - VOTE +1 pending that change.

File: gremlin-python/src/main/python/gremlin_python/process/graph_traversal.py [hunk 1]
```diff
@@ -1451,6 +1467,10 @@ def merge_v(cls, *args):
     def min_(cls, *args):
         return cls.graph_traversal(None, None, Bytecode()).min_(*args)
 
+    @classmethod
+    def none(cls, *args):
+        return cls.graph_traversal(None, None, Bytecode()).none(*args)
```
Comment (xiazcy) [inline gremlin-python/src/main/python/gremlin_python/process/graph_traversal.py:1472]: If my understanding is correct, `none()` is a terminal step so there is no use to add it in the `__` class here (which is why it hasn't been in the class).

File: gremlin-python/src/main/python/gremlin_python/process/graph_traversal.py [hunk 2]
```diff
@@ -2461,6 +2497,8 @@ def where(*args):
 
 statics.add_static('in_e', in_e)
 
+statics.add_static('inV', inV)
```
Comment (vkagamlyk) [inline gremlin-python/src/main/python/gremlin_python/process/graph_traversal.py:2500]: probably not related to this PR

File: gremlin-python/src/main/python/gremlin_python/process/graph_traversal.py (hunk 1 above)
Comment (ryn5) [reply gremlin-python/src/main/python/gremlin_python/process/graph_traversal.py:1472]: Oh right, I forgot that none hasn't been changed to a filtering step yet.  Removing this change.

**Comment by vkagamlyk** [comment]
VOTE+1

**Comment by Cole-Greer** [comment]
Thanks Ryan! LGTM!
VOTE +1

**Pull request merged**